pkg/splunk/controller/configmap.go

```diff
@@ -75,3 +76,53 @@ func GetConfigMapResourceVersion(client splcommon.ControllerClient, namespacedNa
 	}
 	return configMap.ResourceVersion, nil
 }
+
+// GetMCConfigMap gets the MC ConfigMap resource required for that MC
+func GetMCConfigMap(client splcommon.ControllerClient, namespacedName types.NamespacedName) (*corev1.ConfigMap, error) {
+	var configMap corev1.ConfigMap
+	err := client.Get(context.TODO(), namespacedName, &configMap)
+	if err != nil {
+		//if we don't find mc configmap create and return an empty configmap
+		var current corev1.ConfigMap
+		current.Data = make(map[string]string)
```

Comment:
dead line?

Reply:
no, making a map of an empty string and saving that in configmap.data section